src/selfplay/loop.cc

```diff
@@ -123,13 +123,56 @@ void SelfPlayLoop::CmdSetOption(const std::string& name,
 }
 
 void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
+  const int winp1 = info.results[0][0] + info.results[0][1];
+  const int loosep1 = info.results[2][0] + info.results[2][1];
+  const int draws = info.results[1][0] + info.results[1][1];
+
+  // Initialize variables
+  float percentage = -1;
+  float elo = 10000;
+  float los = 10000;
+
+  // Only caculate percentage if any games at all (avoid divide by 0).
+  if ((winp1 + loosep1 + draws) > 0)
+    percentage =
```

Comment:
When entire `if` (or `for`) doesn't fit one line, `{` `}` around body is needed.
( https://google.github.io/styleguide/cppguide.html#Conditionals )

(also other cases like that below)

(but if the body itself is still one line (even if separate), it's fine to omit braces, according to the latest edition of style guide.

Reply:
done